insta-clone/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import PostPage from './components/PostContainer/PostPage';
+import Authentication from './components/Authentication/Authentication';
+
+
+class App extends Component {
+  constructor(){
+    super();
+    this.state = {
+     
+    }
+  }
+
+
+  render() {
+    return <PostPage />
+  }
+}
+
+
+
+export default App;
```

Comment:
Make sure to wrap your App component in the Authentication HOC. Right now the login page is not rendering